frontend/components/Button/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Text from 'root/components/Text';
+
+import './index.css';
+
+const Button = ({ handleClick, href }) => {
+  const Component = href ? 'a' : 'button';
+
+  return (
+    <Component styleName="root" type="submit" href={href} onClick={handleClick}>
+      <Text fontFamily="mono" color="red" weight="bold">
+        #inscreve-te
+      </Text>
+    </Component>
+  );
+};
+
+Button.defaultProps = {
```

Comment:
`defaultProps` after `propTypes` no? Makes more sense to me I at least.

Reply:
If it makes the Doc happier, ok